packages/infra/config/example.ts

```diff
@@ -41,6 +41,10 @@ export const config: EnvConfig = {
     CW_GATEWAY_AUTHORIZATION_CLIENT_ID: "CW_GATEWAY_AUTHORIZATION_CLIENT_ID",
     CW_GATEWAY_AUTHORIZATION_CLIENT_SECRET: "CW_GATEWAY_AUTHORIZATION_CLIENT_SECRET",
   },
+  cqSecretNames: {
```

Comment:
Just to confirm, do we really want to call this `cq` from CareQuality and have the stack called IHE?

Reply:
removing anyway